### DISCUSSION TITLE
Nodule detection running time

### DISCUSSION THREAD

**Discussion by chanuan** [Project-MONAI/tutorials#1911] [Q&A]
Hello, I am using the method provided in https://github.com/Project-MONAI/tutorials/tree/main/detection for lung nodule detection. Can you tell me the configuration and running time when you tested the code? My configuration is 3090 24G. Running one epoch takes about 1000 seconds. However, in a blog from 2022, I saw that he used 3090 24G. Under the same configuration, he runs one epoch in about 500 seconds. Have you updated the detection code during this period? The first one below is my running screenshot, and the second one is the running screenshot of others on the blog.

![image](https://github.com/user-attachments/assets/59016774-a6d0-4746-84ba-4599565950e2)
![image](https://github.com/user-attachments/assets/3f3e8b2b-5521-4d6f-a4ec-beb710a7a827)

**Comment by KumoLiu** [comment]
Hi @chanuan, for this tutorial, we haven't had many updates during this period. In [this section](https://github.com/Project-MONAI/tutorials/tree/main/detection#32-3d-detection-training), we mention that "with a single DGX1V 16G GPU, it took around 55 hours to train 300 epochs for each data fold." However, any changes in training time may also be due to updates from MONAI.

**Reply by chanuan** [reply]
Thanks for the answer
